packages/ember-htmlbars/lib/keywords/get.js

```diff
@@ -22,17 +21,90 @@ function labelFor(source, key) {
   return `(get ${sourceLabel} ${keyLabel})`;
 }
 
+let DynamicKeyStream = BasicStream.extend({
+  init(source, keySource) {
+    assert('DynamicKeyStream error: source must be a stream', isStream(source)); // TODO: This isn't necessary.
+
+    // used to get the original path for debugging and legacy purposes
+    var label = labelFor(source, keySource);
+
```

Comment:
so this extend, isn't aligned with es6 goals. That is fine for now, eventually (one we moved to es6) we can revisit and fix.